plugin/k8s_external/external_test.go

```diff
@@ -147,21 +147,33 @@ var tests = []test.Case{
 			test.SOA("example.com.	5	IN	SOA	ns1.dns.example.com. hostmaster.example.com. 1499347823 7200 1800 86400 5"),
 		},
 	},
+	{
+		Qname: "svc11.testns.example.com.", Qtype: dns.TypeA, Rcode: dns.RcodeSuccess,
+		Answer: []dns.RR{
+			test.A("svc11.testns.example.com.	5	IN	A	1.2.3.4"),
+		},
+	},
+	{
+		Qname: "svc12.testns.example.com.", Qtype: dns.TypeA, Rcode: dns.RcodeSuccess,
+		Answer: []dns.RR{
+			test.CNAME("svc12.testns.example.com.	5	IN	CNAME	dummy.hostname"),
```

Comment:
We should set the test up to expect an `A` record for `dummy.hostname` in the answer also.

Reply:
I've borrowed some code from dns64 testing and added it to k8s_external. Is this what you had in mind?

Reply:
Actually, I think just leave it out for now. I'd rather not introduce an interface just for testing purposes.  Sorry about that.